clients/oraide-vscode/src/extension.ts

```diff
@@ -46,6 +46,16 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 
     start(context, serverPath, workspaceFolderPath, gameFolderPath!);
+
+    // Detect if a file identified as YAML is actually MiniYAML and switch if so.
+    vscode.workspace.onDidOpenTextDocument(e => {
+        if (e.languageId == 'yaml') {
+            const text = e.getText();
+            if (text.includes('\t')) {
```

Comment:
I wonder if this is even necessary, considering there shouldn't be any "real" yaml files inside an OpenRA workspace.(?)

Reply:
Well, there are a few, e.g. GitHub workflow files and tool configs. It's a conservative approach that tries to be minimally disruptive. If someone uses spaces for indentation of MiniYAML then they can still manually switch the language per file, or alternatively assign all '.yaml' files through the workspace `settings.json`:

```json
    "files.associations": {
        "*.yaml": "miniyaml"
    }
```

Reply:
CI is configured using YAML yaml.

Reply:
Ah good point.